src/client/components/scroll-container/scroll-container.tsx

```diff
@@ -0,0 +1,39 @@
+require('./scroll-container.css');
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { $, Expression, Executor, Dataset } from 'plywood';
+import { Stage, Clicker, Essence, DataSource, Filter, Dimension, Measure } from '../../../common/models/index';
+// import { ... } from '../../config/constants';
+import { SvgIcon } from '../svg-icon/svg-icon';
+import { Fn } from "../../../common/utils/general/general";
+
+export interface ScrollContainerProps extends React.Props<any> {
+  onScroll: Fn;
+  style?: Lookup<any>;
+  ref?: string;
+  className?: string;
+  onMouseLeave?: Fn;
+  onMouseMove?: Fn;
+  onClick?: Fn;
+}
+
+export interface ScrollContainerState {
+}
+
+export class ScrollContainer extends React.Component<ScrollContainerProps, ScrollContainerState> {
+
+  render() {
+    const { style, onScroll, onMouseLeave, onMouseMove, onClick } = this.props;
+    return <div
+      className="scroll-container"
```

Comment:
I think the outer component should be called "scroller" and the inner div should be "scroller-inner" or something